software/feather-wiki.yml

```diff
@@ -0,0 +1,17 @@
+name: "Feather Wiki"
+website_url: "https://feather.wiki"
+source_code_url: "https://codeberg.org/Alamantus/FeatherWiki"
+description: "A lightning fast and infinitely extensible tool for creating personal non-linear notebooks, databases, and wikis that is entirely self-contained, runs in your browser, and is only 58 kilobytes in size."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Javascript
+tags:
+  - Wikis
+  - Static Site Generators
+  - Note-taking & Editors
+  - Knowledge Management Tools
```

Comment:
Same as above https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/728. Please remove this tag.